### PR TITLE
Create Base Archive Template on src/render_engine/render_engine_templ…

### LINKED ISSUE #104: Create Base Archive Template
Currently you have to provide a base template... There should be a simple collection title with an archive.

```jinja
<h1>{{collection_title}}</h1>

{% for page in pages %}
<div><a href="{{page.url_for}}">{{page}}</a></div>
{% endfor %}
```

This would go in `render_engine_templates`

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
fixed #104 

Hi [kjaymiller](https://github.com/kjaymiller),
I'm trying to add couple of recommended lines on [src/render_engine/render_engine_templates/base.html](https://github.com/kjaymiller/render_engine/blob/main/src/render_engine/render_engine_templates/base.html).

But not sure the file location, please check it out

### CODE REVIEW

**Comment by kjaymiller** [comment]
base isn't where it should go but I'm working on some things that may make this need some more thought... We can leave it open for now but may not wind up being added due to the issue being not relevant.

**Comment by viktoriussuwandi** [comment]
> base isn't where it should go but I'm working on some things that may make this need some more thought... We can leave it open for now but may not wind up being added due to the issue being not relevant.

ok @kjaymiller , I'm open to help

File: src/render_engine/render_engine_templates/base.html
Comment (kjaymiller) [inline src/render_engine/render_engine_templates/base.html:1]: The added content in this file should be in a new file perhaps `archive.html`.

you'd want to extend `base.html`.


```html
{% extends 'base.html' %}
{% block content %}
                <h1>{{collection_title}}</h1>
                {% for page in pages %}
                <div><a href="{{page.url_for}}">{{page}}</a></div>
                {% endfor %}
{% endblock %}
```

Comment (kjaymiller) [reply src/render_engine/render_engine_templates/base.html:1]: @viktoriussuwandi to clarify you'd create this in a new file `src/render_engine/render_engine_templates/archive.html`

**Review by kjaymiller — CHANGES_REQUESTED** [review]

File: src/render_engine/render_engine_templates/base.html
Comment (viktoriussuwandi) [reply src/render_engine/render_engine_templates/base.html:1]: @kjaymiller , 
1. I try put these lines on the new file src/render_engine/render_engine_templates/archive.html
2. I try to restore [src/render_engine/render_engine_templates/base.html](https://github.com/kjaymiller/render_engine/blob/main/src/render_engine/render_engine_templates/base.html).

Please check it out

**Review by kjaymiller — APPROVED** [review]

**Comment by kjaymiller** [comment]
@all-contributors please add @viktoriussuwandi for Code

**Pull request merged**